src/main.zig

```diff
@@ -109,7 +110,7 @@ pub fn serialize(comptime T: type, allocator: Allocator, data: T, list: *ArrayLi
                         var tlist = ArrayList(u8).init(allocator);
                         defer tlist.deinit();
                         for (data) |item| {
-                            try serialize(ptr.child, item, &tlist);
+                            try serialize(ptr.child, allocator, item, &tlist);
```

Comment:
Fixed this extra nit not detected since there wasn't a test running the line, but I've came now to use this line in some phant related stuff.